practice-app/backend/api/migrations/0002_league_season_standing.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.1 on 2023-05-10 13:02
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0001_initial'),
```

Comment:
boş array olmalı sanırım

Reply:
buraları biz yazmıyoruz buraları django kendisi database tableleri inşa ederken yazıyor onun için dokunmayalım

Reply:
Sıkıntı yok en kötü herkesin migrationlarını bi kontrol ederiz en son